Parsers/ASimProcessEvent/Parsers/ASimProcessCreateSentinelOne.yaml

```diff
@@ -0,0 +1,103 @@
+Parser:
+  Title: Process Create ASIM parser for SentinelOne
+  Version: '0.1.1'
+  LastUpdated: Jul 24, 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.4'
+References:
+- Title: ASIM ProcessEvent Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+- Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM Process Event normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: ASimProcessCreateSentinelOne
+EquivalentBuiltInParser: _Im_ProcessCreate_SentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled: bool=false) {
+      SentinelOne_CL
+      | where not(disabled) 
+          and event_name_s == "Alerts."
+          and alertInfo_eventType_s == "PROCESSCREATION" 
+      | project-rename
+          DvcId = agentDetectionInfo_uuid_g,
+          EventStartTime = sourceProcessInfo_pidStarttime_t,
+          TargetProcessCommandLine = targetProcessInfo_tgtProcCmdLine_s,
+          TargetProcessId = targetProcessInfo_tgtProcPid_s,
+          TargetProcessName = targetProcessInfo_tgtProcName_s,
+          EventUid = _ResourceId,
+          TargetProcessCreationTime = targetProcessInfo_tgtProcessStartTime_t,
+          DvcHostname = agentDetectionInfo_name_s,
+          ActingProcessName = sourceProcessInfo_name_s,
+          ParentProcessName = sourceParentProcessInfo_name_s,
+          ActingProcessCommandLine = sourceProcessInfo_commandline_s,
+          ActingProcessGuid = sourceProcessInfo_uniqueId_g,
+          ActingProcessSHA1 = sourceProcessInfo_fileHashSha1_s,
+          ParentProcessSHA1 = sourceParentProcessInfo_fileHashSha1_s,
+          ActingProcessSHA256 = sourceProcessInfo_fileHashSha256_s,
+          ParentProcessSHA256 = sourceParentProcessInfo_fileHashSha256_s,
+          DvcOs = agentDetectionInfo_osName_s,
+          DvcOsVersion = agentDetectionInfo_version_s,
+          TargetProcessIntegrityLevel = targetProcessInfo_tgtProcIntegrityLevel_s,
+          EventOriginalType = alertInfo_eventType_s,
+          EventOriginalUid = alertInfo_dvEventId_s
+      | extend
+          ActingProcessId = sourceProcessInfo_pid_s,
+          ActorUsername = coalesce(sourceProcessInfo_user_s, 'N/A'),
```

Comment:
always ActorUsername and TargetUsername are same?

Reply:
As discussed, we are getting user info related to sourceProcess only.